qutebrowser/completion/completionwidget.py

```diff
@@ -377,3 +377,17 @@ def completion_item_del(self):
         if not index.isValid():
             raise cmdexc.CommandError("No item selected!")
         self.model().delete_cur_item(index)
+
+    @cmdutils.register(instance='completion', hide=True,
+                       modes=[usertypes.KeyMode.command], scope='window')
+    def completion_item_yank(self, sel=False):
+        """Yank the current completion item onto the clipboard.
+
+        Args:
+            sel: True to use the primary selection instead of the clipboard.
```

Comment:
The "True to" part won't really make sense when reading this as documentation for `--sel` as an user. I'd just say "Use the ..." here.